### ISSUE TITLE
New board SDRPi request a branch.

### ISSUE THREAD

**Issue by HexSDR** [open-sdr/openwifi#143]
Hi ,
I have designed a zynq + ad9361 board which runs openwifi well .Hareware feature is : ZYNQ XC7Z020CLG400 ,1GB DDR3 memory fo PS, 1G ETHER RJ45 for PS,1G  ETHER RJ45 for PL, USB OTG(act as USB host), USB uarts for PS and PL.on board USB to JTAG debuger,TF card , bootable QSPI FLASH and also 27 IO from PL bank; AD9361 RF design is based FMCOMMS3 with TX amplifier. 

These features listed above is similar as ZEDBOARD+FMCOMMS3.For the applications as CSI data dumping ,I also designed an UDP stack accompanied by this PL ethernet port on board .Using this method I will try to modify some source code to ofload the bandwidth and CPU usage,making a more clear and smart way to capture raw IQ data from RF chip .

This SDRPi which size is as small as  Raspberry Pi runing openwifi can be embedded in user application such as UAV,IoT,WIFI router... I request a new branch for this board named as SDRPi.
Thanks.

**Comment by JiaoXianjun** [comment]
Looks promising!
Already manufactured? Could you attach a photo?

We are preparing a big release. Will be public soon.
After that, I will create a branch for your contribution.

**Comment by HexSDR** [comment]
This is the previous version of a demo board. This current board  modified slightly in progressing will be available in about one month.

![KE)T$SAU0G6$}M) $L(69RF](https://user-images.githubusercontent.com/96402566/158101932-0bf8ece3-7041-4fcd-beaf-d4b50712b13f.png)

![GY77N4`PR(MPUEU9QX(9DC5](https://user-images.githubusercontent.com/96402566/158102027-1d0f71aa-aff9-49a7-8d08-adf198d2c705.png)

**Comment by JiaoXianjun** [comment]
Hello,

We have just published our new openwifi release openwifi-1.3.0-wilsele.
I also created sdrpi branch for openwifi and openwifi-hw:
https://github.com/open-sdr/openwifi/tree/sdrpi
https://github.com/open-sdr/openwifi-hw/tree/sdrpi

You can put your changes needed by sdrpi board by creating pull request from your own branch (please create) to sdrpi branch. Then we can review and discuss on sdrpi branch.

**Comment by JiaoXianjun** [comment]
More important, before contribution, could you please check https://github.com/open-sdr/openwifi/blob/master/CONTRIBUTING.md

and sign the CLA. Then send it to Filip.Louagie@UGent.be, xianjun.jiao@ugent.be and Ingrid.moerman@ugent.be ?

Thanks!

**Comment by HexSDR** [comment]
Thanks ,
I have signed this CLA and mailed to you.  I will upload my design later.

**Comment by catkira** [comment]
SDRPi looks nice, is it already available somewhere? Is it commercial or open source?

**Comment by JiaoXianjun** [comment]
Indeed, if there is website/github for sdrpi, where people could check schematic/pcb design, that would be great.

**Comment by HexSDR** [comment]
It has already been designed OK and  now under producting ,for some reasons the producting processing is so slow ,I am very sorry for this delay.  
A github for sdrpi is under developing ,some more thing will put on it . and also will update to this sdrpi branch. In my opinion, it may not open all source files , but will absolutely be sufficent for sdrpi user .

**Comment by JiaoXianjun** [comment]
@HexSDR I have checked https://github.com/HexSDR/openofdm_rx/tree/dot11zynq
It is OK.
Would you please make a pull request to https://github.com/open-sdr/openofdm/tree/dot11zynq ?

Thanks!

**Comment by HexSDR** [comment]
OK ， I will try to do it,thanks


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "open-sdr/openwifi"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月8日(星期一) 下午5:36
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [open-sdr/openwifi] New board SDRPi request a branch. (Issue #143)





 
@HexSDR I have checked https://github.com/HexSDR/openofdm_rx/tree/dot11zynq
 It is OK.
 Would you please make a pull request to https://github.com/open-sdr/openofdm/tree/dot11zynq ?
 
Thanks!
